src/services/findUsedById.js

```diff
@@ -0,0 +1,5 @@
+import users from '../api/users';
+
+export const findUsedById = (todo) => {
```

Comment:
```suggestion
export const findUsedById = (userId) => {
```
such functions should accept `id` as an argument